doc/release_notes/v1.4.0.rst

```diff
@@ -42,3 +42,5 @@ Enhancements
 * Added user warning, or exception in strict mode, if a DICOMDIR has an
   unexpected transfer syntax (:issue:`848`)
 * Handle missing offset tags in DICOMDIR (:issue:`981`)
+* Added optional `handler` argument to
+  :func:`~pydicom.dataset.Dataset.decompress` (:issue:`537`)
```

Comment:
I'd add a little more explanation here: 'This lets you specify a particular handler, rather than following pydicom's default order'.